### ISSUE TITLE
Adding Turkish Support

### ISSUE THREAD

**Issue by yapanel** [feldberlin/timething#27]
## What

I like to add Turkish alignment support. There are already retrained good Wave2Vec2 models we can use.

## Why

However, new models do not seem to have a PIN associated with them. And I struggled to find how to extract this information. Not sure what would be the optimal solution here.

## Acceptance Criteria

- [ ] First user facing thing that needs to be implemented
- [x] Second user facing thing that needs to be implemented

**Comment by purzelrakete** [comment]
@yapanel can you explain what you mean with an associated PIN?

**Comment by isolveit-aps** [comment]
> [@yapanel](https://github.com/yapanel) can you explain what you mean with an associated PIN?

@purzelrakete : I believe @yapanel is referring to the models.yaml file. I wanted to add Faroese (carlosdanielhernandezmena/wav2vec2-large-xlsr-53-faroese-100h), and I have the same PIN problem. Where would I find it?

**Comment by yapanel** [comment]
Yes, it was asking for a model pin which I did not understand and hugging
face has stopped issuing the PINs at that point.
Thank you
